head.html

```diff
@@ -9,7 +9,7 @@
 <script defer src="https://www.adobe.com/etc.clientlibs/globalnav/clientlibs/base/feds.js" id="feds-script"></script>
 <script defer src="https://static.adobelogin.com/imslib/imslib.min.js"></script>
 <script defer type="module" src="/scripts.js"></script>
-
+<script async src="https://api.at.getsocial.io/get/v1/7a87046a/gs_async.js"></script>
```

Comment:
What is this ?

Reply:
Oops, that snuck in there by accident, good catch! It's the GetSocial include which will be handled in #165. I'll remove it.